symplyphysics/laws/kinematic/vector/acceleration_of_transfer_between_relative_frames.py

```diff
@@ -0,0 +1,101 @@
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    add_cartesian_vectors,
+    subtract_cartesian_vectors,
+)
+
+# Description
+## Suppose two reference frames, one of which is fixed (S) and the other one is moving (S'). The movement of
+## a body stationary in moving frame S' due to the movement of the frame itself is called transfer movement.
+## The acceleration related to such movement is called transfer acceleration. It is composed of the acceleration
+## of the moving frame relative to the fixed frame, centripetal acceleration and the acceleration due to uneven
+## rotation of the moving frame. The transfer acceleration only depends on the movement of frame S' relative to
+## stationary frame S, so its physical meaning would be that it is the acceleration in S of a point stationary in S'.
+
+# Law: a_tr = a_0 + a_centripetal + a_non_uniform_rotation
+## a_tr - vector of transfer acceleration of body B relative to fixed frame S
+## a_0 - vector of acceleration of moving frame S' relative to fixed frame S
+## a_centripetal - vector of centripetal acceleration of body B in moving frame S'
+## a_non_uniform_rotation - vector of acceleration caused by non-uniform rotation of frame S'
```

Comment:
Is it acceleration of a body? Maybe it's better to name it smth like `a(t)` to indicate that it is not constant?

Reply:
Technically, any of them can be functions of time. And this one _can_ be constant, e.g. if $`\dot w`$ is constant.